app/javascript/packages/document-capture/components/in-person-post-office-outage-alert.tsx

```diff
@@ -0,0 +1,23 @@
+import { t } from '@18f/identity-i18n';
+import { Alert } from '@18f/identity-components';
+import { getConfigValue } from '@18f/identity-config';
+
+function InPersonPostOfficeOutageAlert() {
+  return (
+    <Alert type="warning" className="margin-bottom-4">
+      <>
+        <strong>
+          {t('idv.failure.exceptions.post_office_outage_error_message.post_cta.title')}
```

Comment:
and could we switch `post_office_outage_error_message` to `usps_outage_error_message` everywhere it occurs?

Reply:
Done.